### PR TITLE
Bump the ulimit when running the `tidy` task if needed

### PR DESCRIPTION
**Pull request by FlorentClarret**
### What does this PR do?

Bump the ulimit when running the `tidy` task if needed

### Motivation

Depending on the terminal emulator, people can have various values for the max number of files open. On iterm2 it seems to be 256, which is pretty low for the `tidy` task. 

We bump it for the current process so people don't face the issue and don't need to tweak their own config.

We might move it to the init section so all task can benefit from it if it works well enough but let's keep it for just this one for now.

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

File: tasks/go.py [hunk 1]
```diff
@@ -411,6 +411,16 @@ def tidy_all(ctx):
 
 @task
 def tidy(ctx):
+    if os.name != 'nt':
```
Comment (KevinFairise2) [inline tasks/go.py:414]: What is `nt` ? Might be worth to add a small comment to explain that.

**Review by KevinFairise2 — APPROVED** [review]

File: tasks/go.py (hunk 1 above)
Comment (FlorentClarret) [reply tasks/go.py:414]: Windows, adding it

**Review by amenasria — APPROVED** [review]
LGTM

**Pull request merged**